README.md

```
@@ -1,21 +1,60 @@
+
+**1: Model View Controller (MVC)**
+
+This project uses the MVC pattern to create the GUI for this project. The model code can be found inside the `Models` folder, the views can be found inside the `Views` folder, and the controllers can be found inside the `Controller` folder.
+
+The Problem: We wanted to write maintainable code with good organization, and also build and test components easily so we could divide our tasks among team members.
+
+The Solution: The MVC model fixed our problem. With this method we modularized our project into three sections. This made development easier and faster as teammates could work on diffrent modules. This method also forced use to organization our code.
+
+**2: Singleton**
+
+This project uses the Singleton design pattern to implement a data manager. The code can be found inside `Models/Data_manager.py`, and we use the DataManager in the `Controller/main_controller.py `file.
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)